src/components/Card/README.md

```diff
@@ -436,6 +436,78 @@ Use to be able to use custom React elements as header content.
 </Card>
 ```
 
+### Full example of card
+
+<!-- example-for: web -->
+
+Use as a broad example that includes most props available to card.
+
+```jsx
+<Card
+  secondaryFooterAction={{content: 'Dismiss'}}
+  primaryFooterAction={{content: 'Export Report'}}
+>
+  <Card.Header
+    actions={[
+      {
+        content: 'Total Sales',
+      },
+    ]}
+    title="Sales"
+  >
+    <Popover
+      active={false}
+      activator={
+        <Button disclosure plain>
+          View Sales
+        </Button>
+      }
+      onClose={() => {}}
+    >
+      <ActionList items={[{content: 'Gross Sales'}, {content: 'Net Sales'}]} />
+    </Popover>
+  </Card.Header>
+  <Card.Section>
+    <TextContainer>
+      You can use sales reports to see information about your customers' orders
+      based on criteria such as sales over time, by channel, or by staff.
+    </TextContainer>
+  </Card.Section>
+  <Card.Section title="Total Sales Breakdown">
+    <ResourceList
+      resourceName={{singular: 'sale', plural: 'sales'}}
+      items={[
+        {
+          sales: 'Orders',
+          amount: 'USD$0.00',
+        },
+        {
+          sales: 'Returns',
+          amount: '-USD$250.00',
+        },
+      ]}
+      renderItem={(item) => {
+        const {sales, amount} = item;
+        return (
+          <ResourceList.Item accessibilityLabel={`View Sales for ${sales}`}>
+            <Stack>
+              <Stack.Item fill>{sales}</Stack.Item>
+              <Stack.Item>{amount}</Stack.Item>
+            </Stack>
+          </ResourceList.Item>
+        );
+      }}
+    />
+  </Card.Section>
+  <Card.Section title="Deactivated reports" subdued>
```

Comment:
We talked about removing the subdued section for now

Reply:
I ended up opening another issue and pushed a PR to fix the margin issue [here](https://github.com/Shopify/polaris-react/pull/962).  Once this PR gets approved we can keep the subdued section. Let me know if that works for you!